packages/web-components/src/components/table-of-contents/table-of-contents.ts

```diff
@@ -460,7 +460,11 @@ class DDSTableOfContents extends HostListenerMixin(
         targetY = elem.offsetTop - mobilePadding!;
       }
 
-      window.scrollTo(0, targetY);
+      window.scrollTo({
+        top:targetY,
+        left: 0,
+        behavior:'smooth'
```

Comment:
Another nit pick (sorry).
```suggestion
        behavior: 'smooth'
```